plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -354,6 +360,50 @@ static bool ffmpeg_mux_start(void *data)
 	return true;
 }
 
+static bool ffmpeg_hls_mux_start(void *data)
+{
+	struct ffmpeg_muxer *stream = data;
+	obs_data_t *settings;
+	const char *path_str;
+	const char *stream_key;
+	struct dstr path = {0};
+
+	if (!obs_output_can_begin_data_capture(stream->output, 0))
+		return false;
+	if (!obs_output_initialize_encoders(stream->output, 0))
+		return false;
+
+	settings = obs_output_get_settings(stream->output);
+	obs_service_t *service;
```

Comment:
Move obs_service_t *service; to the beginning where all the variables are declared.